umi_tools/umi_tools.py

```diff
@@ -50,12 +48,22 @@ def main(argv=None):
 
         return
 
-    elif argv[2] in ["--help", "-h", "--help-extended"]:
+    elif len(argv) > 2 and  argv[2] in ["--help", "-h", "--help-extended"]:
```

Comment:
I think this elif can be removed. The tools all have -h/--help/--help-extended options so no need to catch anything here

Reply:
My guess is that this was added because the standard `--help` within each tool doesn't output the version number?

Reply:
Ah, maybe. 

On a related note, the `--version` options for each tool also don't properly return the version. Should probably just remove that option from the tools and make it available via `umi_tools --version` only